module/modmanager/manager.go

```diff
@@ -40,11 +40,12 @@ var (
 // NewManager returns a Manager.
 func NewManager(parentAddr string, logger golog.Logger, options modmanageroptions.Options) modmaninterface.ModuleManager {
 	return &Manager{
-		logger:       logger,
-		modules:      map[string]*module{},
-		parentAddr:   parentAddr,
-		rMap:         map[resource.Name]*module{},
-		untrustedEnv: options.UntrustedEnv,
+		logger:                  logger,
+		modules:                 map[string]*module{},
+		parentAddr:              parentAddr,
+		rMap:                    map[resource.Name]*module{},
+		untrustedEnv:            options.UntrustedEnv,
+		removeOrphanedResources: options.RemoveOrphanedResources,
```

Comment:
Need to guard against this being nil, as it's required for normal functioning.

Reply:
Good point. Guarded here and where it's used.